Header header
int32 ac_id
Trajectory[] trajectories
